koku/api/report/ocp/capacity/cluster_capacity.py

```diff
@@ -9,15 +9,17 @@
 from dataclasses import field
 from decimal import Decimal
 
+from django.db.models import Sum
 from django.db.models.query import QuerySet
 
 
 # Developer Note:
 # This is common code used in the
 # query handler and node capacity.
-def calculate_unused(row, finalized_mapping={}):
+def calculate_unused(row, finalized_mapping=None):
     """Calculates the unused portions of the capacity & request."""
     # Populate unused request and capacity
+    finalized_mapping = finalized_mapping if finalized_mapping else {}
```

Comment:
This eliminates branching:
```suggestion
    finalized_mapping = finalized_mapping or {}
```

```
>>> None or {}
{}
```
```